package-release.sh

```diff
@@ -45,28 +45,30 @@ while [ $# -gt 0 ]; do
 done
 
 function build_arch {
+  local _1="$1"
```

Comment:
Can you give it a better name than _1?

local arch="$1"
shift
local cross_args="$@"

Reply:
Sure thing, although the latter can't be a normal variable as `"$@"` is an array expansion. I'd politely argue that `"$@"` should remain as-is because it's a common shellscript idiom.